README.md

```diff
@@ -596,9 +596,11 @@ layers:
         inherits: 'App\SomeInterface'
 ```
 
-### More Collectors
+### Custom Collectors
 
-As deptrac is in a very early state, feel free to contribute your own collector.
+You can even create custom collectors in your project by implementing the `CollectorInterface`.
```

Comment:
Could you use the FQCN for `CollectorInferface`?

Reply:
done https://github.com/sensiolabs-de/deptrac/pull/320/commits/617a1fd835ffcce455c521c5ef7023fdd659e572